### ISSUE TITLE
Publish tags and consider a new release

### ISSUE THREAD

**Issue by thesamesam** [festvox/speech_tools#51]
Hello,

From https://www.cstr.ed.ac.uk/projects/festival/download.html, it looks like the last release was in 2014.

Would it be possible for speech_tools to create tags for previous releases (or at least new ones going forward) in this repo, but also consider making a new release so distributions can easily pick up the fixes in master?

Thanks!

**Comment by abathur** [comment]
I'd like to see a new release/tags here as well.

> it looks like the last release was in 2014.

FWIW http://festvox.org/packed/festival/2.5/ contains a 2017 release.

> Would it be possible for speech_tools to create tags for previous releases (or at least new ones going forward) in this repo

It looks like the commit history in this repo only goes back to slightly before the 2017 release--I assume this commit or the one after it roughly correspond with the release I linked above:

https://github.com/festvox/speech_tools/commit/0f888b64aae47ad86be5ef3d4e543aea9fc0fdb0

**Comment by UnitedMarsupials** [comment]
No, seriously, how about it? I'm trying to update the [audio/festival FreeBSD-port](https://www.freshports.org/audio/festival), and it'd be much nicer to _not_ have to glue together quite as many different patches...

**Comment by lenzo-ka** [comment]
I need to defer to @awbcmu for this one -- it sounds like a good thing to do
